Makefile

```diff
@@ -381,11 +381,11 @@ all: $(alltests)
 examples : $(allexamples)
 
 # Benchmarks
-allbenchmarks = petsc-bp1 petsc-bp3
+allbenchmarks = petsc-bps
 bench_targets = $(addprefix bench-,$(allbenchmarks))
 .PHONY: $(bench_targets) benchmarks
 $(bench_targets): bench-%: $(OBJDIR)/%
-	cd benchmarks && ./benchmark.sh --ceed "$(BACKENDS)" -r $(*).sh
+	cd benchmarks && ./benchmark.sh --ceed "$(BACKENDS)" -r petsc-bp3.sh
```

Comment:
This should be fixed to run some subset of (petsc-bp1 petsc-bp2 petsc-bp3 petsc-bp4 petsc-bp5 petsc-bp6) as the default for `make benchmark`. The trick is, petsc-bps needs to be made and the petsc-bp1 petsc-bp2 petsc-bp3 petsc-bp4 petsc-bp5 petsc-bp6 scripts need to be run in the benchmark folder.

Reply:
Fixed. Default is `make benchmarks` runs BP1 and BP3 but the script can be called in the benchmarks folder to run the other BPs.